Core/src/org/sleuthkit/autopsy/datasourcesummary/datamodel/DataSourceTopDomainsSummary.java

```diff
@@ -0,0 +1,63 @@
+/*
+ * Autopsy Forensic Browser
+ *
+ * Copyright 2020 Basis Technology Corp.
+ * Contact: carrier <at> sleuthkit <dot> org
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.sleuthkit.autopsy.datasourcesummary.datamodel;
+
+import java.util.Date;
+import java.util.List;
+import java.util.stream.Collectors;
+import java.util.stream.IntStream;
+import org.sleuthkit.datamodel.DataSource;
+
+/**
+ * Provides summary information about top domains in a datasource.
+ */
+public class DataSourceTopDomainsSummary {
+    private static final long SLEEP_TIME = 5000;
+        
+//    private final SleuthkitCaseProvider provider;
+//    
+//    public DataSourceTopDomainsSummary() {
+//        this(SleuthkitCaseProvider.DEFAULT);
+//    }
+//        
+//    public DataSourceTopDomainsSummary(SleuthkitCaseProvider provider) {
+//        this.provider = provider;
+//    }
```

Comment:
Remove the commented out code?

Reply:
Same here.  I can remove or comment this out, but I'm hoping this code will be of use when we have the underlying query.

Reply:
I can take it out for now.